<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gesture</class>
 <widget class="QDialog" name="Gesture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="var1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter the name of the gesture (only one alphabet allowed)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>381</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter the application to be opened by the gesture</string>
   </property>
  </widget>
  <widget class="QComboBox" name="var2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>116</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>firefox</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gedit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>texmaker</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>rhythmbox</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 211, 171);
</string>
   </property>
   <property name="text">
    <string>Add a Gesture</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter the name of the gesture to check</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 211, 171);
</string>
   </property>
   <property name="text">
    <string>Check Gesture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 211, 171);
</string>
   </property>
   <property name="text">
    <string>Detect Gesture</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
